public/js/index.js

```diff
@@ -2864,18 +2865,52 @@ function initTopicbar() {
             throttle: 500,
             cache: false,
             onResponse: function(res) {
-                var formattedResponse = {
+                let formattedResponse = {
```

Comment:
Hmm we don't use `let` anywhere else in index.js. Are you sure that this doesn't change our compatibility?

Reply:
`let` and `const` where introduced with:


Feature | Chrome | Edge | Firefox (Gecko) | Internet Explorer | Opera | Safari
-- | -- | -- | -- | -- | -- | --
Basic support | 41.0 | 12 | 44 (44) | 11 | 17 | ?

* Is there a list what minimal browser versions are targeted by gitea?
* My IDE mark all `var` with warning. 
I use `let` and `const` as they're more specific what type of variable I need. Furthermore I'm sure the variable isn't changed somewhere else in the code. (->which result in better code -> my opinion)

If I should change this back to `var` I'll have to look for these keywords in my other PRs as well.

Reply:
Usage of `let` is fine. see [here](https://github.com/go-gitea/gitea/blob/master/package.json#L13) for supported browsers, basically modern browsers and IE11, but we will probably eventually drop IE11 at some point.

Reply:
> My IDE mark all var with warning.

That sould not happen if it respects `.eslintrc`. Did you run `make npm` and have your editor configured to use eslint?

Reply:
`make js` runs without problems, thanks for the hint with `make npm` -> ide errors are gone now